src/hardware/gameblaster.cpp

```diff
@@ -294,6 +304,8 @@ void GameBlaster::Close()
 	devices[1].reset();
 
 	is_open = false;
+
+	MIXER_UnlockMixerThread();
```

Comment:
Can't we reduce the scope of the lock to encompass only `MIXER_DeregisterChannel`?

Reply:
Maybe.  I was just trying to play it safe with these.  This code only gets executed at startup and shutdown so it doesn't really make a difference if we lock for an extra ms or whatever.

Reply:
Yeah, that makes sense. Best to leave it like this then.